src/os_common/filename.c

```diff
@@ -157,3 +158,45 @@ __wt_copy_and_sync(WT_SESSION *wt_session, const char *from, const char *to)
     __wt_scr_free(session, &tmp);
     return (ret);
 }
+
+/*
+ * __wt_backup_read --
```

Comment:
I don't really think this function belongs inside the library. I don't see any other use for it other than `wtperf`. It should be part of that program as the code and what it is doing is very user-like and not library-like.

Reply:
I have moved this function into misc.c under wtperf folder. Let me know if it is appropriate to access WT_SESSION_IMPL within wtperf.